invenio_records_editor/views.py

```diff
@@ -17,9 +18,9 @@
 def create_editor_blueprint(app):
```

Comment:
Question, why this function?
We can just register the Blueprint in the entrypoints, without having to add the registration manually when creating the app.